### PR TITLE
Portfolio branch two

### PR DESCRIPTION
**Pull request by EmmaWenzel**
Made some formatting changes and added a picture to polish the portfolio, I'll deploy it to the line server after this

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -46,23 +46,19 @@ <h3>Here are some places you can find me:</h3>
...
       <p> Seagulls are my second favorite
-           type of bird. This particular seagull is french; we'll call him
+           type of bird. This particular seagull is <i>french</i>; we'll call him
            Louis. I found Louis at Le Mont-Saint-Michel. He posed for my picutre.
         </p>
-      <img src="/images/IMG_5916.JPG" alt="seagull?" width = "300">
-
-      <p>This is my portfolio. Doesn't it look nice?</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
```
Comment (devonwasson) [inline portfolio/src/main/webapp/index.html:14]: seeing as addRandomGreeting() is no longer being used, has it been removed from the javascript files as well?

Comment (EmmaWenzel) [reply portfolio/src/main/webapp/index.html:14]: Nope, but it's done now

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -36,7 +21,7 @@ function addRandomFact() {
       'I can still recite most of Hamlet\'s third soliloquy.', 
       'I have 7 pets.', 
       'I can dance a jig.'];
-
+    console.log(facts[2]);
```
Comment (devonwasson) [inline portfolio/src/main/webapp/script.js:24]: If everything is looking good, this log should be removed. Logging is great for debugging but, once thing are working, we don't want to push debugging to master.

File: portfolio/pom.xml [hunk 3]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (ofk-goog) [inline portfolio/pom.xml:37]: Remember to remove the TODO comment after you've done the thing :)

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (EmmaWenzel) [reply portfolio/src/main/webapp/script.js]: Sounds good, that's removed

File: portfolio/pom.xml (hunk 3 above)
Comment (EmmaWenzel) [reply portfolio/pom.xml]: Thanks! I took it out

**Pull request merged**